Gemfile

```diff
@@ -31,7 +31,7 @@ gem 'sassc-rails'
 gem 'simple_form', '~> 5.0'
 gem 'turbolinks'
 gem 'uglifier', '>= 1.3.0'
-gem 'webpacker', '~> 3.5'
+gem 'webpacker', '>= 4'
```

Comment:
I think it would be good to have this be limited to the 4.x series, because major version upgrades usually need to be done intentionally or they will break something.

I tried upgrading webpacker to 4.x myself, without carefully following the instructions (or knowing how to properly follow them, truthfully), and it didn't appear to work as intended.

Suggested: `'webpacker', '~> 4'` (or `'webpacker', '~> 4.0'` effectively the same.)

(You may know this already, but IMO it's handy and worth sharing to everyone: `bundle lock` will quickly update your lockfile without needing to actually download/compile/install gems.)